### ISSUE TITLE
FGDCurve missing inspector

### ISSUE THREAD

**Issue by S4TURN21** [Facepunch/sbox-issues#1881]
I have tried to create my own asset with FGDCurve type property, but its inspector is missing.

It used to look like this:

![image](https://user-images.githubusercontent.com/84143424/171033698-3140b77f-5843-486b-ad4e-15a08eb094c5.png)

**Comment by S4TURN21** [comment]
FGDCurve still missing its inspector, but now you can just use Curve after Garry updated it. It works even better than the FGDCurve inspector.

**Comment by robotboy655** [comment]
Yeah will probably get rid of FGDCurve in favor of the new stuff, just need to get the new editor working in Hammer/ModelDoc
